grassroots-backend/src/auth/GoogleOAuth.strategy.ts

```diff
@@ -39,6 +39,7 @@ export class GoogleOAuthStrategy extends PassportStrategy(
       clientSecret,
       callbackURL,
       scope: ["email", "profile"],
+      prompt: "login",
```

Comment:
Without this, the logout button pretends to work, but then you're still authed with Google and it silently logs you back in whenever necessary.